great_expectations/core/freshness_diagnostics.py

```diff
@@ -18,59 +17,52 @@
 
 
 @dataclass
-class AddedDiagnostics:
+class FreshnessDiagnostics:
```

Comment:
"Freshness" is more appropriate moving forward as it also implicitly encapsulates "addition" of an object to a store